common/build/build-common/api-extractor-base-esm.json

```diff
@@ -0,0 +1,15 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "./api-extractor-base.json",
+	"mainEntryPointFilePath": "<projectFolder>/lib/index.d.mts",
+	"dtsRollup": {
+		"enabled": true,
+		"alphaTrimmedFilePath": "<projectFolder>/lib/<unscopedPackageName>-alpha.d.mts",
+		"betaTrimmedFilePath": "<projectFolder>/lib/<unscopedPackageName>-beta.d.mts",
+		"publicTrimmedFilePath": "<projectFolder>/lib/<unscopedPackageName>-public.d.mts",
+		"untrimmedFilePath": "<projectFolder>/lib/<unscopedPackageName>-untrimmed.d.mts"
+	},
```

Comment:
For Jason's information, where is internal? How are packages going to negotiate things behind the scenes? Those are all `alpha` even if we never are working to make them public?

Reply:
The ./internal export will use untrimmed types (or the standard index.d.ts). https://github.com/microsoft/FluidFramework/pull/18899 has a ton of examples.

Sorry I didn't quite understand your other questions.

Reply:
The other questions are moot since there will be an internal entrypoint.